### PR TITLE
Fix - user must be of type string

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Some old deprecated user columns had some constrains, forcing the data to be strings. With the recent changes, we were storing them as simple arrays, and this is conflicting with its constrains.

## Addresses
- https://github.com/Budibase/budibase/issues/13684
- [BUDI-8251 - user must be of type string](https://linear.app/budibase/issue/BUDI-8251/user-must-be-of-type-string)

## Launchcontrol
Fixing a bug causing save issues with old deprecated single columns

### CODE REVIEW

File: packages/server/src/sdk/app/rows/search.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export async function search(
 
   const table = await sdk.tables.getTable(options.tableId)
   options = searchInputMapping(table, options, {
-    isSql: !!table.sql || !!env.SQS_SEARCH_ENABLE,
```
Comment (mike12345567) [inline packages/server/src/sdk/app/rows/search.ts:83]: Why the removal of this?

**Review by mike12345567 — APPROVED** [review]
LGTM

**Comment by mike12345567** [comment]
Addresses: https://github.com/Budibase/budibase/issues/13684

**Review by mike12345567 — APPROVED** [review]
LGTM - nice improvements, making the bulk import API better!

**Comment by adrinr** [comment]
During these changes, we realised that bulk import (used for tests) was not treating the input row with functions like bbprocessor. This was causing some tests to not work as expected

**Pull request merged**